src/server/goodid/veramo.js

```diff
@@ -0,0 +1,81 @@
+import { once } from 'lodash'
+
+// Core interfaces
+import { createAgent } from '@veramo/core'
+
+// Core identity manager plugin
+import { DIDManager, MemoryDIDStore } from '@veramo/did-manager'
+
+// Ethr did identity provider
+import { KeyDIDProvider } from '@veramo/did-provider-key'
+
+// Core key manager plugin
+import { KeyManager, MemoryKeyStore, MemoryPrivateKeyStore } from '@veramo/key-manager'
+
+// Custom key management system for RN
+import { KeyManagementSystem } from '@veramo/kms-local'
+
+// W3C Verifiable Credential plugin
+import { CredentialPlugin } from '@veramo/credential-w3c'
+
+// Custom resolvers
+import { DIDResolverPlugin } from '@veramo/did-resolver'
+import { Resolver } from 'did-resolver'
+import { getResolver as keyDidResolver } from 'key-did-resolver'
+
+import MultiWallet from '../blockchain/MultiWallet'
+
+const getSeedFromPrivateKey = (privateKeyHex, bytes = 64) => {
```

Comment:
I dont understand this method, why is it required?

Reply:
veramo requires 64 **bytes** (128 hex chars) value as pkey seed, our wallet have 32 **bytes** (64 hex chars) pkey
this method concatenates a few pkey values to obtain 128 hex chars exactly

actually it's enough to combine 2 private keys, so it could be simplified - see https://github.com/GoodDollar/GoodServer/pull/455/commits/aa2cb1c02fbb4049830474269910aa23689a4663